package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "react-jsonschema-form",
+  "name": "@rjsf/core",
```

You probably don't want to change this. It should be the name of the repo if it's a monorepo.
```suggestion
  "name": "react-jsonschema-form",
```

---

Done

---

What do you think should be done with the `version` attribute? I don't think Lerna automatically bumped it upon release, and I'm not sure if `version` is even relevant in the top of the monorepo.